.github/workflows/release-chectl.yml

```diff
@@ -60,9 +60,20 @@ jobs:
               # or just pass it in
               workflow_id=$this_workflow_id
 
-              curl -sSL https://api.github.com/repos/${this_repo}/actions/workflows/${workflow_id}/dispatches -X POST -H "Authorization: token ${GITHUB_TOKEN}" -H "Accept: application/vnd.github.v3+json" -d "{\"ref\":\"master\",\"inputs\": {\"${this_var}\":\"${this_val}\"} }" || die_with "[ERROR] Problem invoking action https://github.com/${this_repo}/actions?query=workflow%3A%22${this_action_name// /+}%22"
+              inputsJson="{}"
+
+              IFS=',' read -ra paramMap <<< "${this_params}"
+              for keyvalue in "${paramMap[@]}"
+              do 
+                  key=${keyvalue%=*}
+                  value=${keyvalue#*=}
+                  echo $var1
+                  inputsJson=$(echo "${inputsJson}" | jq ". + {\"${key}\": \"${value}\"}")
+              done
+
+              https://api.github.com/repos/${this_repo}/actions/workflows/${workflow_id}/dispatches -X POST -H "Authorization: token ${this_github_token}" -H "Accept: application/vnd.github.v3+json" -d "{\"ref\":\"${workflow_ref}\",\"inputs\": ${inputsJson} }" || die_with "[ERROR] Problem invoking action https://github.com/${this_repo}/actions?query=workflow%3A%22${this_action_name// /+}%22"
```

Comment:
this random URL here looks like a copy and paste error

Reply:
also might be good to break this super long line with some `\` to make it more legible.